draft-trammell-rains-protocol.md

```diff
@@ -688,18 +689,11 @@ the entire Message, generated as in {{signatures}}, and to be verified against
 an infrastructure key (see {{obj-infrakey}}) for the RAINS Server originating
 the message.
 
-A Message map MAY contain a capabilities (1) key, for exposing capabilties of
-the message sender. The first Message sent from one peer to another MUST contain
+The value of the capabilities key, when present, is an array of Capabilities or
+the hash thereof. The first Message sent from one peer to another MUST contain
 the capabilities key. The capabilities mechanism is described in
 {{capabilities}}.
 
-A Message map MUST contain a token (2) key, whose value is a 16-byte array.
```

Comment:
Where did these go?

Reply:
I moved it up where the other Message elements are mentioned